wger/core/api/serializers.py

```diff
@@ -91,8 +91,36 @@ class UserCreationSerializer(serializers.ModelSerializer):
     '''
     Create User Serializer
     '''
+    email = serializers.EmailField(max_length=255, required=False)
+    email = serializers.EmailField(max_length=255, required=False)
```

Comment:
Good work. Why do we have two email serializer definitions here?

Reply:
That's good @wekesa931, we have agreed since its just a typo error, @kburudi  will change it in his branch.

Reply:
It's okay. The feature works well

Reply:
Kindly remove one of the email serializer definitions. This will allow easier tracking of the tasks to have all the implementation on one PR.